Content.Server/Tabletop/Components/TabletopGameComponent.cs

```diff
@@ -13,6 +15,18 @@ public class TabletopGameComponent : Component
     {
         public override string Name => "TabletopGame";
 
+        [DataField("boardName")]
+        public string BoardName { get; } = "tabletop-default-board-name";
+
+        [DataField("setup", required: true)]
+        public TabletopSetup Setup { get; } = new TabletopChessSetup();
+
+        [DataField("size")]
+        public Vector2i Size { get; } = (274 + 64, 274);
```

Comment:
Just set the chess prototype to (338, 274) and set the default to a square. Ultimatelly this will be gone as all viewports will be the same size, as you're looking at the table instead of a specific game.

Reply:
Changed it to the arbitrary value of 300, 300